apps/react-18-tests-v9/src/App.cy.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { mount } from '@cypress/react';
+
+import { App } from './App';
+
+const providerSelector = '.fui-FluentProvider';
+
+describe('App with React 18', () => {
+  describe('FluentProvider', () => {
+    it('should apply matching className in strict mode', () => {
+      mount(
+        <React.StrictMode>
+          <App />
+        </React.StrictMode>,
+      );
+
+      cy.get(providerSelector)
```

Comment:
curious what's the purpose of testing this on both e2e(cypress) and unit(jest) level @TristanWatanabe @micahgodbolt  as it creates duplication. 

Also I'm not sure about robustness of this test. what has that className to do with react 18 compatibility ? 

Note - same questions for v8 react-18-tests

Reply:
I would guess that it's currently a simple to to check that the app does not crash (i.e. provider exists). Honestly I would try to leverage our existing cypress tests instead 🤔